package.json

```
@@ -24,9 +24,11 @@
 		"margin",
 		"padding"
 	],
+	"peerDependencies": {
+    "tailwindcss": ">=3.0.0"
+  },
```

After testing it seems that version `^2.0.0` works anything lower failed so we can probably include version 2 for now. Not exactly sure if we want `^` vs `>=`, however.

```suggestion
	"peerDependencies": {
		"tailwindcss": "^2.0.0 || >=3.0.0"
	},
```

---

Well done! I pushed the changes to the PR. `^2.0.0` should work with all minor and patches from version 2. `>=3.0.0` should even allow upcoming major versions of Tailwind. Even Tailwind does this in their official plugins, so we should be safe to go here. 🙂